docs/volume/volume-snapshots.md

```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 5
+sidebar_label: Volume Snapshots
+title: ""
+keywords:
+- Volume Snapshot
+- Volume Snapshots
+Description: Take a snapshot for a volume from the Volume page.
+---
+
+# Volume Snapshots
+
+## Create Volume Snapshots
+
```

Comment:
may add a note that a recurring snapshot is currently not supported and is tracked via https://github.com/harvester/harvester/issues/572

Reply:
Added